### ISSUE TITLE
go 1.19 not installed, cannot renovate any go1.19 repos

### ISSUE THREAD

**Issue by sp3nx0r** [whitesource/renovate-on-prem#218]
We're running whitesource/renovate:2.4.1 docker image and running into this error message 
```
Command failed: go mod tidy
go: go.mod file indicates go 1.19, but maximum version supported by tidy is 1.18
```

![Screenshot from 2022-08-04 15-56-45](https://user-images.githubusercontent.com/10695887/182951210-e62113f0-de6d-47c0-be72-09cc186c3369.png)

Our `go.mod` file indeed contains the line `go 1.19` and looks like 1.18.3 is installed on the current image. 

Is there an image update needed to be able to support this? Suppose we could `install-tool golang 1.19.0` as a workaround (untested yet tho).

**Comment by rarkins** [comment]
Yes, you can use `install-tool golang 1.19.0` on top of the existing image for now. Managers will be updated in the next release, coming in 1-2 weeks time.

**Comment by rarkins** [comment]
Go has been updated in https://github.com/whitesource/renovate-on-prem/releases/tag/2.5.0
